d.html.markdown

```diff
@@ -95,16 +97,14 @@ enum Day {
     Wednesday,
     Thursday,
     Friday,
-    Saturday,
```

Comment:
Well... I guess most compilers wouldn't complain about the trailing comma. But since it is of no use, I feel it is better not to have it at all. This is particularly true in case of enums like in this example (days of the week) where no new fields are going to be added. If we were dealing with an enum that one could expect to grow, then I may be ok with that trailing comma because a new field added to the end wouldn't cause the previous line to change (thus marginally simplifying a review). What do you think?